skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -512,9 +512,10 @@ struct espm::Browser::Impl
   }
 };
 
-espm::Browser::Browser(void* fileContent, size_t length)
+espm::Browser::Browser(const void* fileContent, size_t length)
   : pImpl(new Impl)
 {
+  // TODO (in this PR): get rid of casts to non-const stuff
```

Comment:
If it's hard to clean up everything we could ignore that. Otherwise, let's fix.

Reply:
Yeah, sounds not so easy, leaving as is for now, probably I'll make a separate PR for that later